code/modules/awaymissions/gateway.dm

```diff
@@ -216,6 +216,11 @@ var/obj/machinery/gateway/centerstation/the_gateway = null
 		return
 	if(!stationgate || QDELETED(stationgate))
 		return
+	if(isliving(AM))
+		var/mob/living/L = AM
+		if(L.block_gateway)
+			visible_message("<span class='warning'>[src] rejects [L] from entering!</span>")
+			return

```

Comment:
Instead of this, why not implant the simple mob wizards with exile implants? You can put implants in simple animals now.